routing/turns.cpp

```diff
@@ -22,8 +22,9 @@ namespace
 {
 /// The order is important. Starting with the most frequent tokens according to
 /// taginfo.openstreetmap.org we minimize the number of the comparisons in ParseSingleLane().
-array<pair<LaneWay, char const *>, static_cast<size_t>(LaneWay::Count)> const g_laneWayNames = {
-    {{LaneWay::Through, "through"},
+array<pair<LaneWay, char const *>, static_cast<size_t>(LaneWay::Count) + 1> const g_laneWayNames = {
+    {{LaneWay::None, ""}, // +1 to account for this extra notation for none
```

Comment:
I suspect that this line is not needed. We already have LaneWay::None below. Or put the existing line on top.

Reply:
UPD: Existing but with empty string ""

Reply:
This change was needed because `none` can be written as `"none"` or as `""` ([wiki](https://wiki.openstreetmap.org/wiki/Key:turn#Values)). This seems like the best solution, because `""` is a name for a lane, even though it refers to the same thing as another name. Without one of these two, we'd miss some of the lanes.

Reply:
Ok, put this comment to the code :)

Reply:
Fair enough :) It's documented now.